test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -10,6 +10,25 @@
 
 public class UtilsTest {
 
+    @Test
+    public void isAnyNull() {
```

Comment:
This shouldn't be part of this PR. Please create a clean PR everytime you try to add/make a new change.